src/test/txpackage_tests.cpp

```diff
@@ -568,6 +587,205 @@ BOOST_FIXTURE_TEST_CASE(package_witness_swap_tests, TestChain100Setup)
         BOOST_CHECK(!m_node.mempool->exists(GenTxid::Wtxid(ptx_parent2_v1->GetWitnessHash())));
         BOOST_CHECK(m_node.mempool->exists(GenTxid::Txid(ptx_parent3->GetHash())));
         BOOST_CHECK(m_node.mempool->exists(GenTxid::Txid(ptx_mixed_child->GetHash())));
+
+        // package feerate should include parent3 and child. It should not include parent1 or parent2_v1.
+        BOOST_CHECK(mixed_result.m_package_feerate != std::nullopt);
+        CFeeRate expected_feerate(1 * COIN, GetVirtualTransactionSize(*ptx_parent3) + GetVirtualTransactionSize(*ptx_mixed_child));
```

Comment:
22bf054eee6bfc7148bb40150d648b0dcf0db44b nit, this is the only non-const `expected_feerate` in the test and it can be const too
```suggestion
        const CFeeRate expected_feerate(1 * COIN, GetVirtualTransactionSize(*ptx_parent3) + GetVirtualTransactionSize(*ptx_mixed_child));
```

Reply:
Done